senders/telegram/send_test.go

```diff
@@ -168,3 +170,50 @@ func TestPrepareAlbum(t *testing.T) {
 		})
 	})
 }
+
+func TestCheckBrokenContactError(t *testing.T) {
+	logger, _ := logging.ConfigureLog("stdout", "debug", "test", true)
+	Convey("Check broken contact error", t, func() {
+		Convey("Nil error is nil", func() {
+			err := checkBrokenContactError(logger, nil)
+			So(err, ShouldEqual, nil)
+		})
+		brokenContactErrorsList := []*telebot.APIError{
+			telebot.ErrNoRightsToSendPhoto,
+			telebot.ErrChatNotFound,
+			telebot.ErrNoRightsToSend,
+			telebot.ErrUnauthorized,
+			telebot.ErrBlockedByUser,
+			telebot.ErrUserIsDeactivated,
+			telebot.ErrBotKickedFromGroup,
+			telebot.ErrBotKickedFromSuperGroup,
+		}
+		Convey("Broken contact error is properly recognized", func() {
+			for _, brokenContactError := range brokenContactErrorsList {
+				err := checkBrokenContactError(logger, brokenContactError)
+				So(err, ShouldHaveSameTypeAs, moira.SenderBrokenContactError{})
+				So(err.(moira.SenderBrokenContactError).SenderError, ShouldEqual, brokenContactError)
+			}
+		})
+		otherTelebotErrors := []*telebot.APIError{
```

Comment:
Let's move this into the convey section below.

Reply:
Made the update